### PR TITLE
Automatically set beginning of loop at a GHE

### PR DESCRIPTION
**Pull request by vtnate**
### Any background context you want to provide?
We had hard-coded the beginning of the thermal loop. It was easy to make mistakes and was an extra item that needed manual input. What if we start the loop at _some_ GHE and go from there, instead of specifying an _exact_ GHE?
### What does this PR accomplish?
- Remove hard-coded ghe_start_loop from code, and simply start from the first GHE found and go from there.
- Remove is_ghe_start_loop from demo geojson files
### How should this be manually tested?
Use the test outputs from our CI to generate Modelica models with the GMT, and run them by hand. This will ensure that the loop order is generated correctly and passed to the GMT as expected.

Resolves https://github.com/urbanopt/urbanopt-cli/issues/491

### CODE REVIEW

**Comment by mitchute** [comment]
This PR is 🔥

File: demos/sdk_output_skeleton_2_ghe_staggered/run/baseline_scenario/ghe_dir/sys_params.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
         "insulation_thickness": 0.2,
         "insulation_conductivity": 2.3,
         "diameter_ratio": 11,
-        "roughness": 1e-6,
+        "roughness": 1e-06,
```
Comment (mitchute) [inline demos/sdk_output_skeleton_2_ghe_staggered/run/baseline_scenario/ghe_dir/sys_params.json:85]: @vtnate why does the format on these keep flip-flopping?

Comment (vtnate) [reply demos/sdk_output_skeleton_2_ghe_staggered/run/baseline_scenario/ghe_dir/sys_params.json:85]: Do they? Uuuuuuugh. It's Ruff. I can look into which rule is being hit and turn it off.

**Comment by vtnate** [comment]
> This PR is 🔥

The `_loop_order.json` file isn't being built correctly yet in this new paradigm. I'm working on it.

**Pull request merged**